.rubocop.yml

```diff
@@ -154,3 +157,45 @@ Naming/VariableName:
 
 Naming/VariableNumber:
   Enabled: false
+
+Performance/AncestorsInclude: # new in 1.7
+  Enabled: true
+
+Performance/BigDecimalWithNumericArgument: # new in 1.7
+  Enabled: true
+
+Performance/ConcurrentMonotonicTime: # new in 1.12
+  Enabled: true
+
+Performance/MapCompact: # new in 1.11
+  Enabled: true
+
+Performance/RedundantSortBlock: # new in 1.7
+  Enabled: true
+
+Performance/ReverseFirst: # new in 1.7
+  Enabled: true
```

Comment:
Some of these cops would be good to fix like the `RedundantSortBlock`. Should they go in the todo file instead?

Reply:
My understanding is that the pending cops (including `RedundantSortBlock`) that I've added here in .rubocop.yml are enabled but found no offenses.

I wanted to specify pending cops in .rubocop.yml  that didn't require any changes so that we wouldn't need to come back later to enable them.

Reply:
ah cool, makes sense